app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  scope :update_posts_counter, -> { User.find(3).update(posts_counter: User.find(3).posts.count) }
```

Comment:
- [ ] Great job using a scope here and in other models, but you should understand that we are creating counters for all posts written by different users, and not one user. There are so many Ruby methods you can use to increment the counters. The [icrement](https://apidock.com/rails/ActiveRecord/Base/increment!) function is the best choice for the job.
__HINT:__
```ruby
def update_posts_counter
  post.increment!(:posts_counter)
end
```